### DISCUSSION TITLE
user api to add worker server

### DISCUSSION THREAD

**Discussion by pt-cc** [jhuckaby/Cronicle#701] [Q&A]
Hello, I use centos7.9 to install Version 0.9.40 cronicle， and node version is v16.0.0, I  use Single Primary with Workers mode,copy over your conf/config.json file, and then start the service, but I need to click "Add Server", Can I add server using api ?

![image](https://github.com/jhuckaby/Cronicle/assets/19907364/ab20ac85-48c9-4679-aa54-c754e8a1dafc)

**Comment by jhuckaby** [comment]
Moving to discussion.

**Comment by jhuckaby** [comment]
See https://github.com/jhuckaby/Cronicle/discussions/479
